doc/sphinx/users-guide/vcl-built-in-subs.rst

```diff
@@ -356,6 +356,11 @@ The `vcl_backend_fetch` subroutine may terminate with calling
   ``fetch``
     Fetch the object from the backend.
 
+  ``error(status code, reason)``
+    Transition to :ref:`vcl_backend_error` with ``resp.status`` and
```

Comment:
We are dealing with `beresp.status` in this context.

Reply:
Done